m/mika.md

```diff
@@ -0,0 +1,11 @@
+# Mika sy Davis
+
+![alt text](MIKA.jpg)
+
+This artist started in 2005 and became famous in 2006
+
+**Genre:** Basesa
+
+**Artist homepage:** [MIKA & DAVIS](https://web.facebook.com/mikasydavis/?_rdc=1&_rdr) Mika sy Davis is a folk-fusion group from Madagascar composed of two male singer-songwriters named Mika and Franklin Davis. They perform a contemporary form of the traditional basesa genre of the island's east coast,[1] blended with rock, soul and reggae.[2] The band has been credited with revitalizing and nationally popularizing the basesa genre.[1]
```

Comment:
The bio you entered here should replace the 'This artist started...' bit above.
Make sure to tidy it up so it's readable, all the `[1]`s and `[2]`s need to be removed.

Reply:
I've made the changes that are needed to fix.could you check it plesae